src/dash/models/DashManifestModel.js

```diff
@@ -646,6 +646,10 @@ function DashManifestModel() {
                     voRepresentation.codecs = realRepresentation.codecs;
                     voRepresentation.codecFamily = Utils.getCodecFamily(voRepresentation.codecs);
                 }
+                if (realRepresentation.hasOwnProperty(DashConstants.DEPENDENCY_ID)) {
+                    voRepresentation.dependentRepresentation = new Representation();
+                    voRepresentation.dependentRepresentation.id = realRepresentation[DashConstants.DEPENDENCY_ID].toString();
```

Comment:
I think we should handle the case when `@dependencyId` contains multiple `@id` valeus (as whitespace separated list):

@dsilhavy : What about added this to the capability checks, i.e. rejecting any Representation with more than one entry in this attribute?

Reply:
Thank you, great idea! We would like to reject the representation and log it in the console when `@dependencyId` contains multiple `@id` values as it would most likely indicate an error with the packager. Do you think this would be a good way to approach it?